libs/clist/clist.go

```diff
@@ -315,6 +322,9 @@ func (l *CList) PushBack(v interface{}) *CElement {
 		l.wg.Done()
 		close(l.waitCh)
 	}
+	if l.len == MaxLength {
```

Comment:
Don't you think it's better to use `>=`?

Reply:
Does it make a difference? Maxint is the largest possible value for an int to take on